### PR TITLE
Updated to latest gfx, now using crates.io deps, waiting on piston_window to merge/publish

### LINKED ISSUE #236: gfx_graphics depends on itself
`piston_window` window needs `gfx_graphics` Which means is `gfx_graphics` is broken you can't compile any of the examples. Overriding `gfx_graphics` does not work even via the `paths` variable.

### PR DESCRIPTION
**Pull request by mitchmindtree**
(Blocked by [this](https://github.com/PistonDevelopers/piston_window/issues/36))

I removed the examples to get it published on crates.io (which it is under [`0.1.20`](https://crates.io/crates/piston2d-gfx_graphics)).

Once [this piston_window PR](https://github.com/PistonDevelopers/piston_window/pull/35) merges and gets published, I'll ~~add the examples back in and~~ update to `0.1.21`.

~~Closes 236~~
~~Hmm there are still problems with 236 actually.~~
Closes #236

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,48 +14,28 @@ homepage = "https://github.com/PistonDevelopers/gfx_graphics"
...
-image_gif = ["piston-gfx_texture/image/gif"]
-image_jpeg = ["piston-gfx_texture/image/jpeg"]
-image_png = ["piston-gfx_texture/image/png"]
-image_ppm = ["piston-gfx_texture/image/ppm"]
-image_tga = ["piston-gfx_texture/image/tga"]
-image_tiff = ["piston-gfx_texture/image/tiff"]
-image_webp = ["piston-gfx_texture/image/webp"]
+[dependencies]
+freetype-rs = "0.0.10"
+gfx = "0.6.0"
```
Comment (kvark) [inline Cargo.toml]: Please use `0.6.*`

**Comment by kvark** [comment]
Perhaps, we can wait for piston_window PR before merging this one?

**Comment by mitchmindtree** [comment]
@kvark yeah that's the plan :) sorry, I should have been clearer!

I'll leave an issue for bvssvni in the piston_window repo to publish the latest PR when he's ready (I think he's the only one with auth).

(I'll also fix these versions now)

**Comment by kvark** [comment]
Ok, so we are blocked with this PR until piston_window is published?

**Comment by mitchmindtree** [comment]
@kvark yep

**Comment by mitchmindtree** [comment]
I'm going to move the examples out of this crate into [piston-examples](https://github.com/pistondevelopers/piston-examples) in order to resolve [this issue](https://github.com/PistonDevelopers/gfx_graphics/issues/236#issuecomment-106420578).

**Comment by kvark** [comment]
:+1:

**Comment by mitchmindtree** [comment]
Travis' nightly build is passing and the other isn't starting, going to attempt to publish and then merge.

**Comment by mitchmindtree** [comment]
When trying to publish, I get
```
   Uploading piston2d-gfx_graphics v0.1.21 (file:///Users/Mitch/Programming/Rust/gfx_graphics)
api errors: invalid upload request: ApplicationError("invalid feature name specified: piston-gfx_texture/image/webp")
```

If I remove the features, it seems to work fine. Should I just remove the features?

**Comment by bvssvni** [comment]
@mitchmindtree Just move the features for now. The image library is going through some breaking changes.

**Comment by mitchmindtree** [comment]
`0.1.21` Published :smile:

**Pull request merged**